### ISSUE TITLE
Failed at the fibers@1.0.5 install script

### ISSUE THREAD

**Issue by ChangePlaces** [wekan/wekan-autoinstall#7]
Running the install script gives this error in Ubuntu 16.04

![fibers](https://cloud.githubusercontent.com/assets/302031/23744894/f6a88aea-04ad-11e7-9028-23994fc080ce.png)

googling shows: https://github.com/wekan/wekan/issues/522 which suggests the error only comes about due to the incorrect node version

also as per https://github.com/wekan/wekan/issues/634 FYI, on the 1st October the Node.js Foundation will end support for Node.js v0.10.

**Comment by anselal** [comment]
I have tested the script on a clean Ubuntu 16.04 installation and did not get any errors. Are you running the script as it is or are you trying to install a version of wekan other than 0.10.1 ?

**Comment by ChangePlaces** [comment]
Interesting, this is running inside vmware where the os was installed using vmware's easy install - it collects data from you such as username / password and then installs it all. vmware tools is not installed on the system.

The script was run as is.

**Comment by anselal** [comment]
Besides the errors, is wekan service running ? Can you access it via your browser ?

**Comment by ChangePlaces** [comment]
No, it isn't running. `nmap -p 8080 127.0.0.1` shows the port is closed and a wget `127.0.0.1:8080` fails with a `connection refused` error.

**Comment by anselal** [comment]
Sorry, but I can't recreate the problem. I run the script again on a fresh installation and it competed without any errors. Only some warnings, but wekan is running normally.

**Comment by ChangePlaces** [comment]
is this in docker or a vm?

**Comment by anselal** [comment]
I am testing the script using Vagrant with Virtualbox as a provider. So it is a VM

**Comment by ChangePlaces** [comment]
I'm using the server version of ubuntu.

**Comment by anselal** [comment]
I am using the "Official Ubuntu 16.04 LTS (Xenial Xerus) Daily Build" version. There is no server version for 16. But I tried it with server version 12 and 14 and it works fine. You can try to install npm and node.js manually, or you can try our Virtualbox images. You can find them at https://github.com/wekan/wekan/wiki/virtual-appliance

**Comment by anselal** [comment]
I just saw that wekan updated to version 0.12. I will update the script to suit that version. Maybe you can give it a try then.

**Comment by ChangePlaces** [comment]
This is the download link for the version i used: http://releases.ubuntu.com/16.04.2/ubuntu-16.04.2-server-amd64.iso

I'm assuming something is installed on desktop, that isn't on server which is assumed / needed by the script.

I also tried it with server version 14.04 with the exact same problem

**Comment by lets00** [comment]
I have the same error when I run this script in a LXC ubuntu 16.04 environment.
My steps:

**Create a container:**
```
# lxc-create -n wekan -t ubuntu
# lxc-attach -n wekan
```

**Update container and download script**
```
# apt update && apt upgrade -y
# git clone https://github.com/wekan/wekan-autoinstall.git
```
**Run script**
```
# cd wekan-autoinstall/
# ./autoinstall_wekan.sh
```
After this, the npm does not install fibers and wekan does not running 
(netstat -tln)
```
Proto Recv-Q Send-Q Endereço Local          Endereço Remoto         Estado       PID/Program name
tcp        0      0 127.0.0.1:27017         0.0.0.0:*               OUÇA       7748/mongod     
tcp        0      0 0.0.0.0:22              0.0.0.0:*               OUÇA       166/sshd        
tcp6       0      0 :::22                   :::*                    OUÇA       166/sshd  
```

**Comment by ChangePlaces** [comment]
Judging by other issues that I've seen here, the wekan team appear to blame "the build environment" and then close the issue. I've since gone elsewhere. If the software is this bad on installing, then there's something very wrong with the software. If the main contributors want to deny it, then that's further issues afaic.

**Comment by xet7** [comment]
@ChangePlaces 

Currently wiki is editable by any contributor. Somebody seems to want to install Wekan without using sudo, and removes sudo from fibers etc install commands, so it gets broken, as you can see from https://github.com/wekan/wekan/issues/899

I'm thinking of moving documentation to https://readthedocs.org to prevent this happening.

Wekan is currently being ported to Node 1.4 by @datamaker , his in progress repo is at:
https://github.com/datamaker/wekan

I have been making screenshots to Wekan website about features etc that when I build Wekan, it really does work for me.

**Comment by xet7** [comment]
Hmm, moving documentation would be overreaction. I can see from wiki history what has been changed, and fix those :)

**Comment by xet7** [comment]
I'll welcome contributions as always :)

**Comment by anselal** [comment]
@lesilva00 I haven't worked with lxc in the past so I cannot give any feedback, but I think it works the same as docker containers. Since the wekan releases are on 'Fire' lately, and I mean by that that we had 3 releases the past 14 days, I try to keep up with those and update the script.

**Comment by xet7** [comment]
@lesilva00

Wekan is now available as [Ubuntu snap](https://github.com/wekan/wekan-snap)
